src/client/ts/client.ts

```diff
@@ -54,6 +54,12 @@ const setup = (app: State.IApp, socket: SocketIOClient.Socket) => {
     }
   })
 
+  socket.on('gameover', (message: Message.IGameOver) => {
```

Comment:
probably should deny reconnections at this point if we allow them before the game is over